src/System.Collections/src/System.Collections.csproj

```diff
@@ -6,6 +6,8 @@
     <AssemblyName>System.Collections</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
+    <!-- 436 is thrown because BitArray conflict with corelib -->
+    <NoWarn>0436</NoWarn>
```

Comment:
Opened https://github.com/dotnet/corefx/issues/12678 to clean this up from this and all other places